fixtures/src/components/uncontrolled/checkbox-group.js

```diff
@@ -46,25 +51,30 @@ const UncontrolledCheckbox = () => {
                 <li>value two is checked by default</li>
               </ul>
               <CheckboxGroup
-                name='uncontrolled_checkbox_group_default' label='Uncontrolled Example default'
-                onChange={ onChange }
-                onBlur={ onBlur }
+                id='uncontrolled_checkbox_group_default'
+                label='Uncontrolled Example default'
+                defaultValue={ ['value one'] }
```

Comment:
is having `defaultValue` appliable to `CheckboxGroup` is still really needed? Since we're applying directly `onChange` and `onBlur` on `Checkboxes` then why not just apply `defaultChecked` to the wanted `Checkboxes` ? Also it would mean that  mapping children in `CheckboxGroup` is not needed anymore.

Reply:
I agree :thumbsup: Your comment aligns with @jamime's [comment](https://github.com/Sage/carbon/pull/2431#pullrequestreview-322898567) too:

> We don't want to support a controlled checkbox-group at this time. We want users to control each checkbox instead. Please can you remove the code relating to `defaultChecked` and `value`. I think we can remove the `React.Children.map` entirely.

So I definitely think the `defaultValue` prop needs to be removed from `<CheckboxGroup>` here :thumbsup:

@jamime: Once I've removed it, then what should we do about [the **"Default Checked"** section of the **"Uncontrolled Checkbox Group"** fixture](https://github.com/Sage/carbon/blob/47483ff/fixtures/src/components/uncontrolled/checkbox-group.js#L48-L80)? I don't think that section is needed, because the `<CheckboxGroup>` will not be supporting "default checked" functionality itself. That functionality is being provided/demonstrated by [the **"Default Checked"** section of the **"Uncontrolled Checkbox"** fixture](https://github.com/Sage/carbon/blob/f5d07c3/fixtures/src/components/uncontrolled/checkbox.js#L30-L44) for individual `<Checkbox>` components.

Reply:
agreed, you can remove it.

Reply:
@jamime: Thanks, removed in 7bb7919 :thumbsup: